src/components/Node/Node.tsx

```diff
@@ -121,6 +121,7 @@ const Node: FC<NodeProps & { ref: Ref<HTMLDivElement> }> = memo(
                 isGuidance={isGuidance}
                 documentation={documentation}
                 isAccepted={isAccepted}
+                isCurrentNode={isCurrentNode}
```

Comment:
We're passing this around alot. It seems like it is computed in the Graph component. The evaluatedPathway is available through the pathway context so I think we should be able to compute this in the ExpandedNodeMemo instead of passing it all the way through.

Reply:
Hmm so I looked into this. We do have access to the pathway from context but we don't have access to the `nodeName`(which seems like we need to calculate `isCurrentNode`). We do have access to the state label and it seems like `nodeName` is just a trimmed version of the label but I would rather not make that assumption and just pass the the `isCurrentNode` prop. What do you think?

Reply:
Yea you're right. The `nodeName` is the key which can actually be anything. Lets just leave it as you have it.